sv_inline.h

```diff
@@ -1007,6 +1007,30 @@ Perl_sv_setpv_freshbuf(pTHX_ SV *const sv)
     return SvPVX(sv);
 }
 
+/*
+=for apidoc newSVsv
+=for apidoc_item newSVsv_flags
+=for apidoc_item newSVsv_nomg
+
+These create a new SV which is an exact duplicate of the original SV
+(using C<sv_setsv>.)
+
+They differ only in that C<newSVsv> performs 'get' magic; C<newSVsv_nomg> skips
+any magic; and C<newSVsv_flags> allows you to explicitly set a C<flags>
+parameter.
+
+=cut
+*/
+
+PERL_STATIC_INLINE SV *
+Perl_newSVsv_flags(pTHX_ SV *const old, I32 flags)
+{
+    if (!old)
+        return NULL;
+
+    return sv_freshcopy_flags(old, flags);
+}
```

Comment:
It seems to me that sv_freshcopy_flags() could be newSVsv_flags_NN() instead, given how trivial the differences between newSVsv_flags() and sv_freshcopy_flags() are.

Reply:
I did call it exactly that at one point! Might have moved away from it because of handling the `sv_mortalcopy` behaviour too, but I'm happy to rename it back.